geowebcache/s3storage/src/test/java/org/geowebcache/s3/OfflineS3BlobStoreIntegrationTest.java

```diff
@@ -16,15 +16,14 @@
 import org.geowebcache.storage.StorageException;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Offline integration tests for {@link S3BlobStore}.
  *
  * <p>
  */
-@Ignore // this test fails very often on the AppVeyor build and frequently on Travis, disabling
+// this test fails very often on the AppVeyor build and frequently on Travis, disabling
```

Comment:
This comment can be removed, we are not using AppVeyor/Travis any longer and the @Ignore has been removed.